lib/msf/core/exploit/smb/client/psexec.rb

```diff
@@ -1,5 +1,9 @@
 # -*- coding: binary -*-
 require 'rex/proto/dcerpc/svcctl'
+require 'windows_error'
+require 'windows_error/win32'
```

Comment:
If you have already required 'windows_error', I believe that would inherently require 'windows_error/win32'